### PR TITLE
NFC  - Changed release notes info about Jira to GitLab

### PR DESCRIPTION
**Pull request by cmtool**
I'd love some feedback on exactly which release the issues dropped Jira/started using GitLab, as I don't know that information.

Overview
----------------------------------------
_I removed the references to Jira from the release notes file, as Jira has been deprecated. I substituted with references to GitLab, but don't have the exactly release number that the switch occurred. If someone has those numbers/info, please feel free to edit this PR and add those release numbers._

Before
----------------------------------------
_The current status. Please provide screenshots or gifs ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)) where appropriate._

After
----------------------------------------
_What has been changed. Please provide screenshots or gifs ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)) where appropriate._

Technical Details
----------------------------------------
_If the PR introduces noteworthy technical changes, please describe them here. Provide code snippets if necessary_

Comments
----------------------------------------
_Anything else you would like the reviewer to note_

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
add to whitelist

**Comment by eileenmcnaughton** [comment]
@agh1 any comments

**Comment by agh1** [comment]
Thanks for your attention to this detail @cmtool!  The reference to the starting version should stay as-is: it's just to say that prior versions lack release notes.

As far as Jira to GitLab transition, that took place gradually.  There was a period where you could open tickets in either place, then you could only open ones in GitLab but still update in Jira, and then the current freeze of Jira.  I don't think it's important to document the date(s) of the transition, especially since in the context of one issue, it'll be clear which system has the relevant ticket.

Instead of this change, maybe just replace Jira references with "GitLab and Jira".

File: release-notes.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
 # Release Notes
 
-These release notes are manually compiled from pull requests and Jira issues
-starting with CiviCRM 4.7.14.
+These release notes are manually compiled from pull requests and CiviCRM GitLab issues
```
Comment (cmtool) [inline release-notes.md]:
```suggestion
These release notes are manually compiled from pull requests, Jira and GitLab issues
```

File: release-notes.md [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+starting after March 31 2018.
```
Comment (cmtool) [inline release-notes.md]:
```suggestion
starting with CiviCRM 4.7.14.
```

File: release-notes.md [hunk 3]
```diff
@@ -1,18 +1,22 @@
...
 
 Other resources for identifying changes are:
 
-* The Jira project management system at https://issues.civicrm.org
+* The CiviCRM GitLab project management system at https://lab.civicrm.org/groups/dev/-/issues
```
Comment (cmtool) [inline release-notes.md]:
```suggestion
* The CiviCRM GitLab project management system at https://lab.civicrm.org/groups/dev/-/issues
* The Jira project management system at https://issues.civicrm.org
```

File: release-notes.md [hunk 4]
```diff
@@ -13,6 +14,10 @@ Other resources for identifying changes are:
     * https://github.com/civicrm/civicrm-drupal
     * https://github.com/civicrm/civicrm-joomla
     * https://github.com/civicrm/civicrm-wordpress
+    
+These release notes were manually compiled from pull requests and Jira issues
```
Comment (cmtool) [inline release-notes.md]:
```suggestion
```

File: release-notes.md [hunk 5]
```diff
@@ -13,6 +14,10 @@ Other resources for identifying changes are:
...
+starting with CiviCRM 4.7.14. Jira issue creation at https://issues.civicrm.org was
```
Comment (cmtool) [inline release-notes.md]:
```suggestion
```

File: release-notes.md [hunk 6]
```diff
@@ -13,6 +14,10 @@ Other resources for identifying changes are:
     * https://github.com/civicrm/civicrm-drupal
     * https://github.com/civicrm/civicrm-joomla
     * https://github.com/civicrm/civicrm-wordpress
+    
+These release notes were manually compiled from pull requests and Jira issues
+starting with CiviCRM 4.7.14. Jira issue creation at https://issues.civicrm.org was 
+disabled on March 31 2018.
```
Comment (cmtool) [inline release-notes.md]:
```suggestion
```

File: release-notes.md [hunk 7]
```diff
@@ -13,6 +14,7 @@ Other resources for identifying changes are:
     * https://github.com/civicrm/civicrm-drupal
     * https://github.com/civicrm/civicrm-joomla
     * https://github.com/civicrm/civicrm-wordpress
+
```
Comment (cmtool) [inline release-notes.md]:
```suggestion
```

File: release-notes.md [hunk 8]
```diff
@@ -1,10 +1,11 @@
 # Release Notes
 
-These release notes are manually compiled from pull requests and Jira issues
+These release notes are manually compiled from pull requests, Jira and GitLab issues
```
Comment (cmtool) [inline release-notes.md]:
```suggestion
These release notes are manually compiled from pull requests, GitLab and Jira issues
```

**Comment by cmtool** [comment]
@agh1 Thank you for the feedback! I reverted back to most of the original wording, just adding the following:

1. GitLab and Jira issues
2. The link to the GitLab issues (https://lab.civicrm.org/groups/dev/-/issues), similar to the link to the Jira issues.

Are there other changes you'd suggest that I make? Thank you for looking this PR over!

**Comment by eileenmcnaughton** [comment]
@cmtool ideally you would now squash your commits into one - see https://docs.civicrm.org/dev/en/latest/tools/git/#rebasing-interactive

Since you are a new contributor I'm OK to do a squash & merge if that seems daunting but it gives a less-clear history in git

**Comment by agh1** [comment]
@cmtool great--you might add "legacy" to the line linking to Jira so that it reads "the legacy Jira project management system..."

Right now, you have nine commits in this PR, and most of them cancel each other out.  For all of us following along at home, it's valuable to have commits be logical units of change rather than each adjustment you saved.  Consequently, this PR should probably consist of one commit.

Judging from the branch name, it may be that you're making edits within GitHub.  It's really hard to do a lot of git operations within GitHub, so I think you'll have to do this on your computer.  However, this will be a good learning activity, and since it's a simple change that you could re-create from memory, the stakes are low if something goes wrong!

Please forgive me if this is repeating things you're already familiar with.  Here goes:

1. [Install Git.](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)

2. Clone [your fork](https://github.com/cmtool/civicrm-core) of civicrm-core.

   > git clone https://github.com/cmtool/civicrm-core.git

   Your fork will be a remote called `origin`.  You'll default to the `master` branch.

3. Add the main CiviCRM fork as another remote called `upstream`.

   > git remote add upstream https://github.com/civicrm/civicrm-core.git

   > git fetch upstream

   You've now got your local repository on your computer, and it knows about two remotes, `origin` (your fork you can write to) and `upstream` (CiviCRM's fork that you'll want to be merged to).  When you "fetch" a remote, you're getting up-to-date information from it.

4. Check out the branch you've been working on, which happens to be called `patch-1`.

   > git checkout patch-1

5. Tell Git that you'd like your `patch-1` branch to keep track of the `master` branch at the `upstream` remote.

   > git branch --set-upstream-to upstream/master

   Now, if you run `git status`, Git will tell you that your branch and `upstream/master` have diverged, with a certain number of commits each.  That's fine--and since this PR says there are no conflicts with the base branch, these commits are probably touching different files from each other.

6. Rebase interactively so you can squash commits.  "Rebase" means having Git apply your commits to the most recent commit of the base branch rather than the older one you had been on.  The `-i` means that it'll display all your commits first so you can choose what to do with each one.

   > git rebase -i

   You'll see a list of your 9 or 10 commits so far, each with the word `pick`, the short commit ID, and the commit message.  If you were to save as-is, it would keep all the commits as-is--that's what `pick` means.

   Instead, leave the `pick` in the first line, but replace `pick` with `squash` on each of the subsequent commits.  Save and exit the text editor, and it will work through your commits, smashing them together into a single one, itself applied to the most recent commit on `master`.

   You'll be given the opportunity to edit the commit message for the combined commit.  It will start will all of the messages, and you can probably delete a lot of them.  If you were to have used `fixup` instead of `squash`, it would have skipped this step and just used the first commit's message.

  Save the message and exit the editor.  Now you have one commit.

7. Force-push the commit back to your branch

   > git push -f origin patch-1

   Ordinarily, you don't need the `-f`, but since you're pushing the squashed change to your branch that has all of the old commits.

**Comment by agh1** [comment]
Ha--so @eileenmcnaughton said it way more succinctly in the time it took for me to start writing, go to a meeting, and finish it.

**Pull request merged**